src/components/MultiSelect/MultiSelectMenuButton.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import Button from "./../Button/Button";
+import Icon from "./../Icons/Icon";
+import { Box, useMultiStyleConfig } from "@chakra-ui/react";
+import { SelectedItems } from "./MultiSelectTypes";
+
+export interface MultiSelectMenuButtonProps {
+  /** The id of the multiSelect using this button. */
+  multiSelectId: string;
+  /** The label of the multiSelect using this button. */
+  label: string;
```

Comment:
If the above is `multiSelectId`, then this should be `multiSelectLabel`. So that's why I think the name can be `id` instead of `multiSelectId`.

Reply:
fixed